source/projects/lorawan/en/index.md

```diff
@@ -0,0 +1,113 @@
+
+
+## Explore LoRaWan together with the CityLAB
+
+The last two editions of the so called "Open Workshop" (German: *Offene Werkstatt*) at the [CityLAB Berlin](https://www.citylab-berlin.org/) were dedicated to the topic of **GPS Tracking with LoRaWan**. Participants were equipped with our [instructions on GitHub](https://github.com/technologiestiftung/werkstatt) as well as with the necessary hardware, software and other components like the [Dragino LoRaBee](https://www.dragino.com/products/lora/item/109-lora-bee.html) to transfer location data via the LoRa-Net.
+
+In the most recent workshop, we built an optimized tracker. In contrast to the LoRa nodes, this tracker featured individual components that were put together, leading to a more robust (thanks to soldering) as well as compact version of the tracker.
+
+
+### A little preview
+{% include macro-image-section-markdown-small.html src="../images/Nano_Xbee_Image.png" caption="GPS-LoRa-Tracker, as it was built at the Offene Werkstatt (© TSB)"%}
+
+<!-- <img src="../images/Nano_Xbee_Image.png" style="margin-left:6%; margin-right:4%; float:right; width:60%" alt="GPS-LoRa-Tracker CityLAB"> -->
+
+The following hardware components are required for this LoRa GPS tracker:
+* 9V batteries + charger
+* Battery clip 
+* GPS module with antenna 
+* Arduino Nano V3 (Clone)
+* Dragino LoRa Bee 868 (European frequency)
+
+Added up, these parts amount to **just 30€** per tracker – far below the price of the normal GPS trackers, which usually start at 50€.  
+
+As soon as all these components are soldered together, the hardware part is finished. On the software side, it's now necessary to register the tracker in **[The Things Network (TTN)](https://thethingsnetwork.org)** and through that assign to our device a corresponding device address, as well as a network session key and an app key. We'll need these three keys for the program code (also called *Sketch* under Arduino), which will be loaded onto the Arduino Nano V3 microcontroller. Detailed instructions for **how to register your end node in the TTN console**, including screenshots of the user interface, can be found [here](https://www.thethingsnetwork.org/docs/devices/registration.html#personalize-device-for-abp) at The Things Network.
+
+Once the device is registered, you can get **the aforementioned three variables** (device address, network session key and app key) from the TTN console. You can find the Sketch (i.e., the program code) [here on GitHub](https://github.com/technologiestiftung/werkstatt/tree/master/codes_sketches) – name: *GPSTracker_LoRa_Nano_V2.ino*
+
+The following lines need to be adjusted in the code:
+
+```js
+//*** Werte bekommt Ihr aus der TTN-Console, Format msb! ***
+static u1_t NWKSKEY[16] = {
+    0xB2, 0x5F, 0x35, 0x64, 0xB4, 0x89, 0xB8, 0x09, 0x08, 0x12, 0x7D, 0xAC, 0x0F, 0xC6, 0xF1, 0x5C
```

Comment:
Would look a little nicer if you make blocks like 

```c
0xB2, 0x5F,
0x35, 0x64,
0xB4, 0x89,
0xB8, 0x09,
//…
```

Reply:
alright, changed to groups of four